src/main/java/com/powsybl/openloadflow/OpenLoadFlowProvider.java

```diff
@@ -162,6 +162,7 @@ public static AcLoadFlowParameters createAcParameters(Network network, MatrixFac
         LOGGER.info("Connected component mode: {}", parameters.getConnectedComponentMode());
         LOGGER.info("Voltage per reactive power control: {}", parametersExt.isVoltagePerReactivePowerControl());
         LOGGER.info("Reactive Power Remote control: {}", parametersExt.hasReactivePowerRemoteControl());
+        LOGGER.info("Shunt voltage control: {}", parameters.isSimulShunt());
```

Comment:
A comment for the loadflow api: "simulShunt" is poorly named...
For a user point of view, what does "simulate shunts" means? It does not appear clearly that we are talking about shunt voltage control.

Reply:
Yes, you are right. I have created an issue : https://github.com/powsybl/powsybl-core/issues/1929